subset/switches/test_switch

```diff
@@ -8,8 +8,14 @@ if [ -n "$LOCAL_IP" ]; then
     ip addr add $LOCAL_IP dev `hostname`-eth0
     echo Switch test with port $SWITCH_PORT.
     ping -n -c 10 $SWITCH_IP
-    java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT
+    java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT true
```

Comment:
@ewRedstone In this context, 'true' doesn't actually describe if switchtest runs poe tests. Currently, I have to look in the Main.java of switchtest to see what 'true' or 'false' actually does. Can we have a variable (like $SWITCH_IP or $SWITCH_PORT) that is set to true or false e.g. $POE_ENABLED?

Reply:
I was disucssing with @pisuke today about this and we think that it might be a good idea if the module_config.json contains the $POE_ENABLED environment variable. Do you know how to implement this ? I have changed the test argument to accept $POE_ENABLED as the arg